app/helpers/admin/analytics_helper.rb

```diff
@@ -67,25 +67,76 @@ def total_number_of_searches
       total_searches.html_safe
     end
 
+    def avg_number_searches_per_visit
+      avg_searches_per_visit =
+        "#{AhoyQueries.get_total_number_of_searches
+        .div AhoyQueries.get_total_number_of_ahoy_visits}"
```

Comment:
```suggestion
        .div(AhoyQueries.get_total_number_of_ahoy_visits)}"
```
This is just to make more clear that this is a division, for a few seconds I thought that we were talking about a `<div>` html tag 😅